iml-manager-client/src/lib.rs

```diff
@@ -20,6 +26,8 @@ impl std::fmt::Display for ImlManagerClientError {
             ImlManagerClientError::Reqwest(ref err) => write!(f, "{}", err),
             ImlManagerClientError::InvalidHeaderValue(ref err) => write!(f, "{}", err),
             ImlManagerClientError::UrlParseError(ref err) => write!(f, "{}", err),
+            ImlManagerClientError::SerdeJsonError(ref err) => write!(f, "{}", err),
+            ImlManagerClientError::RunStratagemValidationError(ref err) => write!(f, "{}", err),
```

Comment:
I think this should live under `ImlManagerCliError` instead actually. It seems to specific for the client at this point.

Reply:
I'm not sure I understand. Do I need to remove the `RunStratagemValidationError` type?

Reply:
Move the validation error type into `ImlManagerCliError`